onnxmltools/convert/common/model_util.py

```diff
@@ -14,6 +14,137 @@
 onnx_integer_types = [onnx_proto.TensorProto.UINT8, onnx_proto.TensorProto.INT8, onnx_proto.TensorProto.UINT16,
                       onnx_proto.TensorProto.INT16, onnx_proto.TensorProto.INT32, onnx_proto.TensorProto.INT64]
 
+onnx_operator_version_info = {
```

Comment:
There is another table similar to this under make_node that contains a list of operators that should be set to ai.onnx.ml for the domain name. Is there anyway that the make_node code could use this version info structure instead? This would reduce duplication.

Reply:
You're right. Let me try to move this into converters and make things cleaner.